sources/packages/backend/apps/api/src/testHelpers/auth/institution-auth-helpers.ts

```diff
@@ -137,7 +149,7 @@ export async function authorizeUserTokenForLocation(
     institution.id,
     user.id,
     location.id,
-    InstitutionUserTypes.user,
+    institutionUserType ?? InstitutionUserTypes.user,
```

Comment:
A comment is missing for the new parameter.
Besides that, not a blocker but please consider adding the new parameter using the options.

Reply:
I can see the options added but it is still missing the comment for the `institutionUserType` under it.